### PR TITLE
packages yum: support CentOS 8

### PR DESCRIPTION
**Pull request by kenhys**
Because of some *-devel packages are missing, just disable them for a
while.

### CODE REVIEW

**Comment by kou** [comment]
We should use `dnf` instead of `yum`.

File: configure.ac [hunk 1]
```diff
@@ -1416,6 +1416,7 @@ AM_CONDITIONAL([CENTOS_INIT_PLATFORM],
                 test "${package_platform}" = "centos6"])
 AM_CONDITIONAL([CENTOS_SYSTEMD_PLATFORM],
                [test "${package_platform}" = "centos7" ||
+                test "${package_platform}" = "centos8" ||
                 test "${package_platform}" = "fedora"])
```
Comment (kou) [inline configure.ac]: `test "${package_platform}" != "centos6"`

We need to add this list when CentOS 9 is release if we still use this logic.

File: packages/yum/centos-8/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
+ARG FROM=centos:8
+FROM ${FROM}
+
+ARG DEBUG
+
+RUN \
+  quiet=$([ "${DEBUG}" = "yes" ] || echo "--quiet") && \
+  yum install -y https://packages.groonga.org/centos/groonga-release-latest.noarch.rpm && \
```
Comment (kou) [inline packages/yum/centos-8/Dockerfile]: Is this needed?

File: configure.ac (hunk 1 above)
Comment (kenhys) [reply configure.ac]: Thanks, I'll fix this.

File: packages/yum/centos-8/Dockerfile (hunk 2 above)
Comment (kenhys) [reply packages/yum/centos-8/Dockerfile]: No, I'll remove it.

**Comment by kenhys** [comment]
I've fixed them. Thanks!

**Pull request merged**